src/copyrightinserter.ts

```diff
@@ -97,7 +98,14 @@ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU Affero General Public License for more details.
 
 You should have received a copy of the GNU Affero General Public License
-along with this program.  If not, see <https://www.gnu.org/licenses/>.`]
+along with this program.  If not, see <https://www.gnu.org/licenses/>.`],
+
+        ['mpl2', (holder: string, year:string) =>
+`Copyright (C) ${year} ${holder}
```

Comment:
MPL is copyleft license. It does not require and should not have `Copyright...` line.

Reply:
Thanks for the quick review! Although MPL is a copyleft license, it does not preclude copyright ownership for source code. The [MPL 2.0 FAQ](https://www.mozilla.org/en-US/MPL/2.0/FAQ/#apply) page talks about this under Question 4:

> You may also add additional accurate notices of copyright ownership, such as the name of the copyright holder, but this is not necessary.

Reply:
It sounds like your use case utilizes [Q4](view-source:https://www.mozilla.org/en-US/MPL/2.0/FAQ/#apply) clarifications. 
My concern was that the current implementation creates controversy between inserting a copyleft comment and needing extra keystrokes to delete copyright that is not a part of the notice.
I am OK with leaving this notice. However, if there will be requirements to align the text with the standard (and not optional) form of the license, this line will be removed or made opted-in.

Reply:
@minherz that sounds reasonable to me -  a future option to preclude a copyright line would also be applicable to the two GPL licenses as well. Care would have to be taken to rework the `hasCopyright` function though, as it currently uses regex to search for the word `Copyright` (not a blocker, just something to keep in mind).

Thanks for the discussion and merge! I appreciate it.